docs/architecture/auto-app-management.md

```diff
@@ -78,11 +77,14 @@ In the previous example, there was an assumption that every Application managed
 of the Application that provides similar API and event definitions.  
 In case IntegrationSystem supporting many types of Applications, information about Application type should be stored in the Application labels.
 Let assume that IntegrationSystem supports two types of Applications: `ecommerce` and `marketing` and such information
-is stored in label `{integration-system-name}/applicationType`.
-To create an Application of type `ecommerce`, use following mutation:
+is stored in label `{integration-system-name}/application-type`.
+To register an Application of type `ecommerce`, use following mutation:
+
+> **NOTE**: The labeling input of the API is likely to change in the near future.
```

Comment:
- Please move the note to the top (before main header)
- Rephrase this comment that this document contains API changes, which will be introduced in future: register/unregisterIntegrationSystem + input labels
- Add comment to GraphQL API Hardening issue, to adjust the comment once the API changes with `register/unregister` are ready